cfme/tests/containers/test_ad_hoc_metrics.py

```diff
@@ -14,13 +14,16 @@
 @pytest.fixture(scope="function")
 def metrics_up_and_running(provider):
 
-    router = [router for router in provider.mgmt.o_api.get('route')[1]['items'] if
-              router["metadata"]["name"] == "hawkular-metrics"].pop()
-    hawkular_url = router["status"]["ingress"][0]["host"]
-    response = requests.get("https://{url}:443".format(url=hawkular_url), verify=False)
-
-    assert response.ok, "hawkular failed to start!"
-    logger.info("hawkular started successfully")
+    router = [router for router in provider.mgmt.o_api.get('route')[1]['items']
+              if router["metadata"]["name"] == 'hawkular-metrics' or
+              router["metadata"]["name"] == 'prometheus'].pop()
```

Comment:
**REQUIRED:** This could result in an IndexError if there are no routers - We should handle this and skip the test if so.